dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -1003,29 +1003,14 @@ internal open class RumViewScope(
             is android.app.Fragment -> key.activity
             is NavigationViewTrackingStrategy.NavigationKey -> {
                 if (navControllerActivityField == null) {
-                    internalLogger.log(
-                        InternalLogger.Level.WARN,
-                        InternalLogger.Target.TELEMETRY,
-                        "Unable to retrieve the activity field from the navigationController"
-                    )
                     null
                 } else {
                     navControllerActivityField.isAccessible = true
                     navControllerActivityField.get(key.controller) as? Activity
                 }
             }
             else -> null
-        }
-
-        if (activity == null) {
-            internalLogger.log(
-                InternalLogger.Level.WARN,
-                InternalLogger.Target.TELEMETRY,
-                "Unable to retrieve the activity from $key, " +
-                    "the frame rate might be reported with the wrong scale"
-            )
-            return
-        }
+        } ?: return
```

Comment:
I would rather change level to `MAINTAINER` than to delete it (at least for the first entry, in the `NavigationViewTrackingStrategy.NavigationKey` branch). In the end, we still need to address underlying issue, so logging is still useful.